src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1388,6 +1388,29 @@ void CodeGen::genSSEIntrinsic(GenTreeHWIntrinsic* node)
             break;
         }
 
+        case NI_SSE_SetAllVector128:
```

Comment:
This is completely ignoring containment opportunities.

Reply:
The same applies to other code paths below.

Reply:
Let me improve here in a separate PR because the `Avx.SetAllVector256` has the same issue now.